server/data/licenceApiClient.ts

```diff
@@ -79,6 +80,17 @@ export default class LicenceApiClient extends RestClient {
     )) as Promise<LicenceSummary>
   }
 
+  async getlicenceStatistics(startDate: string, endDate: string, user: User): Promise<LicenceStatistics[]> {
+    try {
+      return (await this.get(
+        { path: `/support/licence-statistics?startDate=${startDate}&endDate=${endDate}` },
+        { username: user.username }
+      )) as Promise<LicenceStatistics[]>
+    } catch (error) {
+      return error.status >= 400 && error.status < 500 ? null : error
```

Comment:
I don't think there are any expected errors for this endpoint so don't need to try/catch

The other calls wrap as absent OMU is modelled as a 404.

Reply:
ok